README.yaml

```diff
@@ -0,0 +1,133 @@
+  - name: "Latest Release"
+    image: "https://img.shields.io/github/release/cloudposse/terraform-aws-documentdb-cluster.svg"
+    url: "https://github.com/cloudposse/terraform-aws-documentdb-cluster/releases/latest"
+  - name: "Slack Community"
+    image: "https://slack.cloudposse.com/badge.svg"
+    url: "https://slack.cloudposse.com"
+
+related:
+  - name: "terraform-aws-rds"
+    description: "Terraform module to provision AWS RDS instances for MySQL or Postgres"
+    url: "https://github.com/cloudposse/terraform-aws-rds"
+  - name: "terraform-aws-rds-cluster"
+    description: "Terraform module to provision an RDS Aurora cluster for MySQL or Postgres"
+    url: "https://github.com/cloudposse/terraform-aws-rds-cluster"
+  - name: "terraform-aws-rds-cloudwatch-sns-alarms"
+    description: "Terraform module that configures important RDS alerts using CloudWatch and sends them to an SNS topic"
+    url: "https://github.com/cloudposse/terraform-aws-rds-cloudwatch-sns-alarms"
+
+# Short description of this project
+description: |-
+  Terraform module to provision an [`Amazon DocumentDB`](https://aws.amazon.com/documentdb/) cluster.
+
+# How to use this project
+usage: |-
+
+  **IMPORTANT:** Do not pin to `master` because there may be breaking changes between releases. Instead pin to the release tag (e.g. `?ref=tags/x.y.z`) of one of our [latest releases](https://github.com/cloudposse/terraform-aws-documentdb-cluster/releases).
+
+  For a complete example, see [examples/complete](examples/complete)
+
+  ```hcl
+  module "documentdb_cluster" {
+    source                  = "git::https://github.com/cloudposse/terraform-aws-documentdb-cluster.git?ref=master"
+    namespace               = "eg"
+    stage                   = "testing"
+    name                    = "docdb"
+    cluster_size            = "2"
+    admin_user              = "admin1"
+    admin_password          = "Test123456789"
+    instance_class          = "db.r4.large"
+    vpc_id                  = "vpc-xxxxxxxx"
+    subnet_ids              = ["subnet-xxxxxxxx", "subnet-yyyyyyyy"]
+    allowed_security_groups = ["sg-xxxxxxxx"]
+    zone_id                 = "Zxxxxxxxx"
+  }
+  ```
+
+references:
+  - name: db-cluster-create
+    description: Creating an Amazon DocumentDB Cluster
+    url: https://docs.aws.amazon.com/documentdb/latest/developerguide/db-cluster-create.html
+  - name: db-instance-class-specs
+    description: Amazon DocumentDB Instance Class Specifications
+    url: https://docs.aws.amazon.com/documentdb/latest/developerguide/db-instance-classes.html#db-instance-class-specs
+  - name: db-cluster-parameter-group-create
+    description: Creating an Amazon DocumentDB Cluster Parameter Group
+    url: https://docs.aws.amazon.com/documentdb/latest/developerguide/db-cluster-parameter-group-create.html
+  - name: limits
+    description: Amazon DocumentDB Limits
+    url: https://docs.aws.amazon.com/documentdb/latest/developerguide/limits.html
+  - name: documentdb-arns
```

Comment:
we use a programmatic name for repos, but I think for articles it makes more sense to use a title.

Reply:
you mean the same as `description`?
description: Amazon DocumentDB Limits

Reply:
I agree with osterman. I think you should use what you have as descriptions for the titles, and then the descriptions can be longer, like "Official AWS documentation on how to create a DocumentDB cluster" or just a repeat of the title but without title case.